pwiz_tools/Skyline/TestRunner/Program.cs

```diff
@@ -669,7 +669,7 @@ private static string LaunchDockerWorker(int i, CommandLineArgs commandLineArgs,
             testRunnerExe = iRelative != -1
                 ? Path.Combine(@"c:\pwiz", testRunnerExe.Substring(iRelative))
                 : @"c:\pwiz\pwiz_tools\Skyline\bin\x64\Release\TestRunner.exe";
-            var testRunnerCmd = $@"{testRunnerExe} parallelmode=client showheader=0 results=c:\AlwaysUpCLT\TestResults log={testRunnerLog}";
+            var testRunnerCmd = $@"{testRunnerExe} parallelmode=client showheader=0 results=c:\AlwaysUpCLT\TestResults_{i} log={testRunnerLog}";

```

Comment:
I still don't feel this should be necessary.  These folders should not be shared at all even without an identifying index... Unless the identifying index is more about knowing which client ran the test when looking at the logs. Though, it seems like that could be done more directly, since not all errors report a path.

Reply:
It isn't necessary. It just makes for a more informative log.